Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -27,6 +27,36 @@ AZ_DECLARE_BUDGET(Terrain);
 
 namespace Terrain
 {
+    Physics::HeightfieldProviderNotifications::HeightfieldChangeMask TerrainToPhysicsHeightfieldChangeMask(AzFramework::Terrain::TerrainDataNotifications::TerrainDataChangedMask mask)
+    {
+        using AzFramework::Terrain::TerrainDataNotifications;
```

Comment:
[nit] The heightfield notification namespace could be added too to use them in the same way.

`using Physics::HeightfieldProviderNotifications;`